src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -34,7 +34,7 @@ curie_map:
   semapv: https://w3id.org/semapv/vocab/
   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
   sssom: https://w3id.org/sssom/
-  oio: http://www.geneontology.org/formats/oboInOwl#
+#  oio: http://www.geneontology.org/formats/oboInOwl#
```

Comment:
## New bug: Duplicate URI stems @joeflack4 @matentzn 
@matentzn Any ideas are welcome.
<details><summary>Details</summary>
<p>
@joeflack4 Address issue

We have duplicate URI stems for the following CURIEs:
- oio: http://www.geneontology.org/formats/oboInOwl#
- oboInOwl: http://www.geneontology.org/formats/oboInOwl#

I know that there is a way to use `curies` to set up aliases. However I am not sure where the fix for this needs to be; it might need to go in several places:
1. `mondo.sssom.config.yml`: Had the duplicate entry; may/may not need updates
  For now, I commented one entry out hoping for a quick temporary fix. Did not work; errors in OAK (see below).
  Also, I need to figure out if this file should be changes, and if that is within the scope of this PR. Perhaps there is a way for us to update this file so that it is reflective of how we would set up the `Converter`, e.g. we could declare which prefixes are authoritative and which ones are aliases here. And for that, I would need to finally learn a bit more about these [extended prefix maps](https://curies.readthedocs.io/en/latest/struct.html#extended-prefix-maps).
2. SSSOM: Probably doesn't need a PR
  The `mondo-ingest` lexmatch calls SSSOM, which then creates the `Converter` object, but it doesn't set up aliases. Seems like this needs to be updated at some point to handle these situations. This may not be a blocker though. No error is produced; I just do a little patch right now on the metadata that SSSOM is returning back in `mondo-ingest`. That is so far sufficient.
3. OAK: Probably needs a PR
  Where I'm currently stuck (as of Jan 11) is in `lexical_indexer.py`, in `lexical_index_to_sssom()`. At the end, it does `msdf.clean_prefix_map(strict=ensure_strict_prefixes)` and fails because it detects the same duplicate prefixes (`oio` and `oboInOwl`). As with SSSOM, there is a `Converter` object being created her which does not handle prefix aliases. I think I'll likely have to change the code to fix this. I'm currently getting:
```
ValueError: new record already exists and merge=False: {('oboInOwl', 'http://www.geneontology.org/formats/oboInOwl#', 'oio', ''): ['prefix match']}
```

</p>
</details>

Reply:
As for (1), please also note https://github.com/monarch-initiative/mondo-ingest/pull/388

For (3), is there no way we can avoid conflicting prefixes going in, e.g. (1)?

Else I am not sure how to deal with this atm..

Reply:
On #388 I took a look; noted!
For (3), the OAK code. I need more time to look into it. I removed conflicting prefixes from (1) but still got the error.

Reply:
Make sure you try this with all changes in #388 as well! The error may be coming from a different prefix map with the same conflicts.

Reply:
Making sure you saw my comment

Reply:
I saw it, but as I explained in [this comment](https://github.com/monarch-initiative/mondo-ingest/pull/394#discussion_r1456679001) and in other comments in the [other PR](https://github.com/mapping-commons/sssom-py/pull/485), removing the a duplicate prefix from these files won't fix the issue. For clarification, the only config #388 the `mondo.sssom.config.yml`, and I tried deleting the duplicate in there already (as discussed in other PR) and it didn't work.